Explorer/Assets/DCL/Multiplayer/Movement/Systems/RemotePlayersMovementSystem.cs

```diff
@@ -52,12 +56,17 @@ private static void HandleFirstMessage(FullMovementMessage firstRemote, ref Char
 
         [Query]
         [None(typeof(PlayerComponent))]
-        private void UpdateRemotePlayersMovement([Data] float deltaTime, ref CharacterTransform transComp, ref CharacterAnimationComponent anim,
-            ref RemotePlayerMovementComponent remotePlayerMovement, ref InterpolationComponent intComp, ref ExtrapolationComponent extComp, in IAvatarView view)
+        private void UpdateRemotePlayersMovement(
+            [Data] float deltaTime,
+            ref CharacterTransform transComp,
+            ref CharacterAnimationComponent anim,
+            ref RemotePlayerMovementComponent remotePlayerMovement,
+            ref InterpolationComponent intComp,
+            ref ExtrapolationComponent extComp,
+            in SimplePriorityQueue<FullMovementMessage> playerInbox,
```

Comment:
can you put` SimplePriorityQueue<FullMovementMessage> playerInbox` inside `RemotePlayerMovementComponent` instead of adding it as separate component to the entity?